modules/47_regipol/regiCarbonPrice/bounds.gms

```diff
@@ -42,8 +42,10 @@ $ENDIF.CCSinvestment
 
 
 ** Force historical bounds on coal
-    vm_cap.up("2020",regi,"pc","1")$((cm_startyear le 2020) and (sameas(regi,"DEU"))) = 38.028/1000;
-    vm_cap.up("2020",regi,"pc","1")$((cm_startyear le 2020) and (sameas(regi,"UKI"))) = 1.3/1000; !!2019 capacity = 7TWh, capacity factor = 0.6 ->  ~1.35GW -> Assuming no new capacity -> average 2018-2022 = ~ 1GW
```

Comment:
any specific reason why you removed the UKI bound? This is necessary to have better approximation to real world values.